terraform/stacks/tooling/variables.tf

```diff
@@ -147,3 +147,7 @@ variable "billing_bucket" {
 variable "cg_binaries_bucket" {
    default = "cg-binaries"
 }
+
+variable "use_nat_gateway_eip" {
+  default = false
```

Comment:
any reason not to just default this to true and yank the override in the pipeline?